src/bindings/js/node/lib/addon.ts

```diff
@@ -38,6 +38,7 @@ interface Core {
   readModelSync(modelBuffer: Uint8Array, weightsBuffer?: Uint8Array): Model;
   importModelSync(modelStream: Buffer, device: string): CompiledModel;
   getAvailableDevices(): string[];
+  getVersions():{ deviceName: string; version: string }[];
```

Comment:
Misalignment with bindings signature

Reply:
Fixed this in the latest commit. Kindly review it.